packages/terra-date-input/src/DateInput.jsx

```diff
@@ -620,7 +623,7 @@ class DateInput extends React.Component {
           aria-disabled={this.props.disabled}
           aria-invalid={this.props.isInvalid}
           aria-required={this.props.required}
-          aria-describedby={this.hotKeyInstructionId}
+          aria-describedby={`${this.hotKeyInstructionId} ${this.props.errorMessageId}`}
```

Comment:
```suggestion
          aria-describedby={ (customProps['aria-describedby']) ?`${this.hotKeyInstructionId} ${customProps['aria-describedby']}` : this.hotKeyInstructionId}
```
this allow consumer passed aria-describedBy to be passed as well

Reply:
Updated.